clients/fides-js/src/lib/tcf.ts

```diff
@@ -48,7 +48,7 @@ export const generateTcString = async ({
   experience,
   tcStringPreferences,
 }: {
-  tcStringPreferences?: TcfSavePreferences;
+  tcStringPreferences?: EnabledIds;
```

Comment:
the logic in this func should be mostly the same as before—we just get a different input obj, so things are a little different (`EnabledIds` actually maps well to this use case, because, like TC strings, it only stores opt ins)